Marlin/Configuration_adv.h

```diff
@@ -883,6 +900,19 @@
                                               // even if it takes longer than DEFAULT_STEPPER_DEACTIVE_TIME.
   //#define PARK_HEAD_ON_PAUSE                // Go to filament change position on pause, return to print position on resume
   //#define HOME_BEFORE_FILAMENT_CHANGE       // Ensure homing has been completed prior to parking for filament change
+
+  //Blowing 
+  #define ADVANCED_PAUSE_FANSPEED    120         //Blowing to cold the purged filament
+  #define ADVANCED_PAUSE_FAN           0
+  #define ADVANCED_PAUSE_DWELL    4*1000         //Time to cold the purged filament
+  // Spool_Swap
+  #define ADVANCED_PAUSE_SPOOL_SWAP              //Run next tool after filament runout
+  #if ENABLED(ADVANCED_PAUSE_SPOOL_SWAP)
+    #define ADVANCED_PAUSE_SPOOL_SWAP_UNLOAD   10 // Don't eject filament because print continue
+    #define ADVANCED_PAUSE_SPOOL_SWAP_LOAD     10 // Same as FWRETRACT if enabled
+    #define ADVANCED_PAUSE_SPOOL_SWAP_EXTRUDE  20 //Purge length
+  //#define ADVANCED_PAUSE_SPOOL_SWAP_COPY_FLOW   // copy old tool flow to next tool 'Recommended'
+  #endif 
```

Comment:
Swapping spool , don't need ejection of filament , it's why new values are needed